tests/conformance/conformance_test.go

```diff
@@ -32,6 +32,9 @@ import (
 	"github.com/containers/image/v5/types"
 	"github.com/containers/storage"
 	"github.com/containers/storage/pkg/reexec"
+	dockertypes "github.com/docker/docker/api/types"
+	dockerdockerclient "github.com/docker/docker/client"
```

Comment:
dockerdocker?

Reply:
We already have imports named client, docker, and dockerclient, so I'm running out of pseudonyms for them.

Reply:
Well, client's a variable, not a package.